src/mbgl/map/map_options.cpp

```diff
@@ -61,4 +62,13 @@ NorthOrientation MapOptions::northOrientation() const {
     return impl_->orientation;
 }
 
+MapOptions& MapOptions::withSize(Size size_) {
```

Comment:
nit: `size`, no need for trailing `_`

Reply:
Fail to build with linux-gcc4.9 when member name and function parameter name is same.

```
declaration of 'size' shadows a member of 'this' [-Werror=shadow]
 MapOptions& MapOptions::withSize(Size size) {
```